vsts/windowsLinuxAndAndroidBuildMatrixConfig.yaml

```diff
@@ -138,6 +138,94 @@ jobs:
           JAVA_VERSION: $(JAVA_VERSION)
           COMMIT_FROM: $(COMMIT_FROM)
 
+      - task : Docker@2
```

Comment:
These tests will run for android, windows, and linux builds, FYI. Do these steps enable the invalid cert server for those builds as well? Or do we need to duplicate these steps in each of the windows/linux/android builds?

Reply:
Good catch, right now it seems this only works on Linux due to some issue with Docker on Windows.

Reply:
Linux is the only one this will work for now (and we should test locally how well it works for android - it should work but this will be the first we test it). Docker for Windows have a bug that is reported but have no resolution and therefore windows agent in the pipeline will not work. On our local test, there are instruction to use WSL (linux) to run this on our windows laptop.

Also, there is a build.ps script for .NET SDK that only run invalid cert tests (Traits) when it is linux (non-window agents). There isn't an equivalent for that here in the changes.